### ISSUE TITLE
[css-scrollbars][css-scrollbars-1] #1960 was resolved without consensus and ignoring a number of issues

### ISSUE THREAD

**Issue by craigkovatch** [w3c/csswg-drafts#3442]
The draft spec has input only from Mozilla and Microsoft, and ignores basically all of the feedback from #1960 and related issues. How can this be considered a proper way to develop a "standard" on the modern Web?

**Comment by astearns** [comment]
@craigkovatch it will be much more productive if you open issues for the remaining use cases you are concerned with. From the thread you reference, I suggest opening one issue for hover colors (though you should be very specific about why the author needs to supply hover colors over having the browser determine hover colors based on the face and track colors), another for border radii, and a third for whatever dimensions you think are needed that are not covered by the current width discussion in #2966.

Please understand that it is likely and entirely appropriate that we may defer some of these issues to a later draft. Incremental progress and patience are often required for standardization.

**Comment by craigkovatch** [comment]
@astearns The issue was closed completely ignoring what has already been stated. I struggle to understand why the burden is on me, or the several other people who expressed concerns, to re-state those concerns in additional threads.

Ignoring the voices of those who disagreed with the exactly-two authors of the current draft is hardly "incremental progress".

**Comment by tabatkins** [comment]
Note in particular that "consensus" is *among CSSWG members*, not web authors.

Author feedback is a *very important* part of our process (and some members *exclusively* represent author interests), but ultimately there's a **lot** of authors in the world, with diverse needs and opinions, and so we're guaranteed that any decision we make will be very unpopular with some fraction of them.

Scrollbar-related topics took up a substantial amount of time at one of our face-to-face meetings this year, and the fact that some websites were currently using more than two colors for their custom scrollbars was definitely brought up. Ultimately, tho, we had to balance between the competing desires of browsers that allowed *zero* scrollbar customization (such as Safari) and those that allowed *total* scrollbar customization; we determined that the 2-color approach would satisfy most users while still being acceptably simple to the less-customization browsers.
